src/js/components/Menu.js

```diff
@@ -184,6 +214,16 @@ var Menu = React.createClass({
 
     if (! this.state.controlFocused && prevState.controlFocused) {
       this.stopListeningToKeyboard(focusedKeyboardHandlers);
+
+      // This was added because removing the focusedKeyboardHandlers also
+      // removes the `down` and `space` events from activeKeyboardHandlers.
+      // - Maybe this behaivor should be changed in the KeyboardAccelerators
+      // mixin itself?
+      // - Maybe the activeKeyboardHandlers and the focusedKeyboardHandlers
+      // should be added to two different components?
+      if (this.state.active) {
```

Comment:
I changed the way how we manage the states a little bit. I thinks it's solved now (and the code is a little bit simpler too.)